Build/BuildScripts/CodeJam.AppVeyor.NUnit.Tests.ps1

```diff
@@ -1,22 +1,21 @@
 ﻿$include = "*.Tests.dll", "*.Tests.NUnit.dll"
-$includePerfTests = "*.Tests.Performance.dll"
-$exclude = "Experimental\\.*?\\CodeJam.Tests.Performance.dll"
 
 mkdir "$env:APPVEYOR_BUILD_FOLDER\_Results" -ErrorAction SilentlyContinue
 
 $wc = New-Object System.Net.WebClient
 
-$targetsDotNet = "net47","net46","net45","net40","net35","net20"
+#run .net tests
+$targetsDotNet = "net48","net472","net471","net47","net461","net45","net40","net35","net20"
```

Comment:
Do we need a separate net46 ?

Reply:
I'd prefer to keep the list as short as possible. Let's keep the current one and add more frameworks as new targeting issue arrive.